lib/mapObjects/CObjectClassesHandler.cpp

```diff
@@ -200,7 +200,7 @@ void CObjectClassesHandler::loadObjectEntry(const std::string & identifier, cons
 		auto range = legacyTemplates.equal_range(std::make_pair(obj->id, id));
 		for (auto & templ : boost::make_iterator_range(range.first, range.second))
 		{
-			handler->addTemplate(templ.second);
+			handler->addTemplate(const_cast<ObjectTemplate*>(templ.second));
```

Comment:
I see a lot of const_cast in the diff. Maybe better store non-const version?

Reply:
Nah, it was just a spaghetti code. Now templates are const everywhere unless they are explicitely modified at the beginning.